genie-web/src/test/resources/application-integration.yml

```diff
@@ -22,6 +22,8 @@ genie:
       location: "file:///tmp/"
     archive:
       location: /archive/location
+  threshold:
+    used-physical-memory-percent: 100
```

Comment:
Can you add a default value to the ```src/main/resources/application.yml```? We try to put default values in there so that users know most of the properties that are available.